modules/appeals_api/app/swagger/appealable_issues/v0/swagger.json

```diff
@@ -232,14 +217,24 @@
             "content": {
               "application/json": {
                 "examples": {
-                  "decision_review_type must be one of: higher_level_reviews, notice_of_disagreements, supplemental_claims": {
+                  "decisionReviewType must be one of: higher-level-reviews, notice-of-disagreements, supplemental-claims": {
                     "value": {
                       "errors": [
                         {
-                          "title": "Unprocessable Entity",
-                          "code": "unprocessable_entity",
-                          "detail": "decision_review_type must be one of: higher_level_reviews, notice_of_disagreements, supplemental_claims",
-                          "status": "422"
```

Comment:
The format of this error has changed because it's validated via the JSON schema now instead of in code